src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,11 @@
+rootLogger.level = error
```

Comment:
with this change, is the --log-level command line arg / env var still supported?

Reply:
Yes, the --log-level command line arg is still supported and it will override the log4.properties.